catalog/task/run-file-updates/0.1/run-file-updates.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: run-file-updates
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to run file updates
+  params:
+    - name: jsonKey
+      type: string
+      description: The json key containing the file updates
+      default: ".spec.extraData.fileUpdates[]"
+    - name: fileUpdatesPath
+      type: string
+      description: The path to the file containing the file updates
+    - name: request
+      type: string
+      description: Name of the request
+      default: "file-updates"
+    - name: synchronously
+      type: string
+      description: Whether to run synchronously or not
+      default: "true"
+  workspaces:
+    - name: data
+      description: Workspace where the file updates to apply are defined
+  steps:
+    - name: run-script
+      image: quay.io/hacbs-release/release-utils:34901b1694963265ac2dcfecc2f56e15a2a8c7fe
+      script: |
+        #!/bin/sh
+        set -ex
+
+        # Read JSON file into a variable
+        jsonData=$(cat $(workspaces.data.path)/$(params.fileUpdatesPath))
+
+        # Extract the key from the JSON file
+        fileUpdates=$(echo "${jsonData}" | jq -c "$(params.jsonKey)")
+  
+        # Iterate over the extracted array and call the script
+        for item in "${fileUpdates}"
```

Comment:
Now the test is breaking here. My case I was using a single repo, but with the following change it should now work:

`while read item
do
  ...
done <<< "${fileUpdates}"`